joomla/joomla-cms
# [4.0] RTL: fix Searchtools Clear button and Actions dropdown

### Summary of Changes 

As title says

### Testing Instructions
Install a clean 4.0-dev. 
Install Persian language and switch to Persian in back-end

Display Article Manager, select an article, click on the Actions button to see dropdown.
Look at the Searchtools Clear button margins.

Patch and run npm.

Display Installed Languages manager and look at Clear button margins
### Before patch
<img width="917" alt="Screen Shot 2019-10-27 at 10 42 54" src="https://user-images.githubusercontent.com/869724/67632710-6e950e80-f8a7-11e9-8ed4-25d20c685039.png">
<img width="447" alt="Screen Shot 2019-10-27 at 10 42 46" src="https://user-images.githubusercontent.com/869724/67632711-6e950e80-f8a7-11e9-8c21-6709c893c9ed.png">


### After patch
<img width="918" alt="Screen Shot 2019-10-27 at 10 45 02" src="https://user-images.githubusercontent.com/869724/67632718-7d7bc100-f8a7-11e9-80eb-bf644ca76dcf.png">
<img width="444" alt="Screen Shot 2019-10-27 at 10 44 52" src="https://user-images.githubusercontent.com/869724/67632719-7d7bc100-f8a7-11e9-9424-ec11e2fc1318.png">
